### PR TITLE
[T4A2][T17-B1]Yan Nuoyuan

### PR DESCRIPTION
**Pull request by nyannnnnnn**

### CODE REVIEW

**Comment by nyannnnnnn** [comment]
[T4A2][T17-B1]Yan Nuoyuan

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -87,7 +87,7 @@ public String getUserCommand() {
         showToUser("[Command entered:" + fullInputLine + "]");
         return fullInputLine;
     }
-
+
```
Comment (JoelT-92) [inline src/seedu/addressbook/ui/TextUi.java:90]: Avoid formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use sourcetree to look through changes before committing, and discard any unrelated changes.

File: src/seedu/addressbook/storage/StorageFile.java [hunk 2]
```diff
@@ -88,6 +88,9 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         /* Note: Note the 'try with resource' statement below.
          * More info: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
          */
+    	if(!path.toFile().exists()){
+    		System.out.println("You do not have the storage file.");
```
Comment (JoelT-92) [inline src/seedu/addressbook/storage/StorageFile.java:92]: Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.

File: src/seedu/addressbook/parser/Parser.java [hunk 3]
```diff
@@ -42,7 +42,7 @@
      */
     public static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
 
-    public Parser() {}
+    static public Parser() {}
```
Comment (JoelT-92) [inline src/seedu/addressbook/parser/Parser.java:45]: I think you may have misunderstood the instructions. You may wish to reattempt after reading this again : https://github.com/nus-cs2103-AY1617S1/addressbook-level2/blob/master/doc/LearningOutcomes.md#use-class-level-members-lo-classlevel

**Comment by JoelT-92** [comment]
@nyannnnnnn Some comments added. Please close the PR after reading comments. 
1. There is no ready for review comment.
2. Avoid accumulating unrelated changes into a single PR. 
   Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.
3. You may have misunderstood the instructions of activity 2 and 4. Please do read them carefully and reattempt if possible.
   Tip: For activity 2 and 4. you can look at how your classmates did it. A2: https://github.com/nus-cs2103-AY1617S1/addressbook-level2/pull/1220, A4: https://github.com/nus-cs2103-AY1617S1/addressbook-level2/pull/894
